Rainforest Eagle: update model numbers (#42513)
home-assistant/home-assistant.github.io: source/_integrations/rainforest_eagle.markdown

ha_integration_type: integration
---

Integrate energy usage and pricing from the Rainforest Automation's
[Eagle 3](https://www.rainforestautomation.com/rfa-z114-eagle-200-2/),
Eagle-200 and
[Legacy Eagle](https://rainforestautomation.com/support/rfa-z109-eagle-support/)
energy gateways. These devices work over Zigbee Energy Profile to connect to
your smart meter. As such, they do not conenct to your regular Zigbee network
(you don't even need a Zigbee network), but typically will need to be

> provisioned (i.e. connected to your meter) by your utility. You then connect

Reviewer: ```suggestion
your smart meter. As such, they do not connect to your regular Zigbee network
(you don't even need a Zigbee network), but typically will need to be
provisioned (for example, connected to your meter) by your utility. You then connect
```

Author: "i.e." is more accurately rendered as "that is to say", but this works too

the device to your home network and can pull energy usage via the device's
local API. The price will only be included if it is provided by the electricity
meter. If you picked an electricity plan in the app, the price data will not be

> availabl

Reviewer: ```suggestion
meter. If you picked an electricity plan in the app, the price data will not be
available.
```

ha_integration_type: integration
---

> Integrates energy usage and pricing from the Rainforest Automation's [Eagle 3](https://www.rainforestautomation.com/rfa-z114-eagle-200-2/), Eagle-200 and [Legacy Eagle](https://rainforestautomation.com/support/rfa-z109-eagle-support/) energy gateways. These devices work over Zigbee Energy Profile to connect to your smart meter. As such, they do not connect to your regular Zigbee network (you don't even need a Zigbee network as part of your system), but typically will need to be provisioned (that is to say, connected to your meter) by your utility. You then connect the Eagle to your home network and can pull energy usage via the device's local API. The price will only be included if it is provided by the electricity meter; if you picked an electricity plan in the app, the price data will not be available.

Reviewer: ```suggestion
Integrates energy usage and pricing from the Rainforest Automation's [Eagle 3](https://www.rainforestautomation.com/rfa-z114-eagle-200-2/), Eagle-200 and [Legacy Eagle](https://rainforestautomation.com/support/rfa-z109-eagle-support/) energy gateways. These devices work over Zigbee Energy Profile to connect to your smart meter. As such, they do not connect to your regular Zigbee network (you don't even need a Zigbee network as part of your system), but typically will need to be provisioned (that is to say, connected to your meter) by your utility. You then connect the Eagle to your home network and can pull energy usage via the device's local API. The price will only be included if it is provided by the electricity meter. If you picked an electricity plan in the app, the price data will not be available.
```
